src/agent/coverage/src/cobertura.rs

```diff
@@ -176,17 +214,28 @@ mod tests {
                 .attr("hits", "0")
                 .attr("branch", "false"),
         )?;
-        _emitter_test.write(XmlEvent::end_element())?; // line
 
+        _emitter_test.write(XmlEvent::end_element())?; // line
         _emitter_test.write(XmlEvent::end_element())?; // lines
-
         _emitter_test.write(XmlEvent::end_element())?; // class
+        _emitter_test.write(XmlEvent::end_element())?; // classes
+        _emitter_test.write(XmlEvent::end_element())?; // package
+
+        _emitter_test.write(
+            XmlEvent::start_element("package")
+                .attr("name", "C:/Users")
+                .attr("line-rate", "0.00")
+                .attr("branch-rate", "0")
+                .attr("complexity", "0"),
+        )?;
+
+        _emitter_test.write(XmlEvent::start_element("classes"))?;
 
         _emitter_test.write(
             XmlEvent::start_element("class")
-                .attr("name", "0")
+                .attr("name", "C:/Users/file2.txt")
```

Comment:
Does this mean that this will not going to work on Linux ?

Reply:
Good point. This test would only work on Windows. WIll Fix.